wuvt/__init__.py

```diff
@@ -172,6 +172,14 @@ def init_app():
     app.register_blueprint(trackman.bp)
     app.register_blueprint(trackman.private_bp, url_prefix='/trackman')
     app.register_blueprint(trackman.api_bp, url_prefix='/trackman/api')
+    trackman.playlists_cache.init_app(app, config={
+        'CACHE_TYPE': "redis",
+        'CACHE_REDIS_URL': app.config['REDIS_URL'],
```

Comment:
If you intend to separate trackman from wuvt-site, shouldn't they use different redis URLs (i.e. different DBs)? There's a risk of conflicting keys if the apps are developed independently.